### ISSUE TITLE
NameError: name 'ParserElement' is not defined

### ISSUE THREAD

**Issue by Nwinternights** [decalage2/ViperMonkey#27]
Hi,
seems that using latest version of oletools (0.52.dev12) Vmonkey raises a traceback about pyparsing module.
  File "/usr/local/lib/python2.7/dist-packages/vipermonkey-0.5-py2.7.egg/vipermonkey/core/__init__.py", line 91, in <module>
    ParserElement.enablePackrat()
NameError: name 'ParserElement' is not defined
Using older version of oletools everything is ok.
regards

**Comment by SimonHLeBlanc** [comment]
Hi, 
I was going to report the same problem on my side. Guess you're faster ;) 

I am using the latest version of oletools and vipermonkey. I have not tested with a older version of oletool though. I'll let you know if the work around works for me. 

Thanks!

**Comment by mhanne0915** [comment]
Hi guys I'm also encountering this error. Do you find any solution/fix? I'm also trying fixing this error. Ill let you know once I manage to fix it.

**Comment by Saalonso** [comment]
Hi, same problem here. Any update on this issue? Thanks!

**Comment by ResearchRacoon** [comment]
Ran into this issue as well. At the top of the script add "from pyparsing import *" and it appears to work. The ParserElement function is located within pyparsing module.

**Comment by recvfrom** [comment]
It looks like the oletools package has a module named pyparsing which ends up first in the library search path for some reason:

/usr/local/lib/python2.7/dist-packages/oletools/thirdparty/pyparsing/

This version of the library seems to be incompatible with vipermonkey (for one thing, it's __init__.py is empty, which means the `from pyparsing import *` done by vipermonkey won't work, but even if you symlink that libraries' __init__.py to it's pyparsing.py file there are missing classes.)

I'm not sure what the "real" fix, but this worked for me:

In /usr/local/lib/python2.7/dist-packages/vipermonkey/core/__init__.py, make the following change (diff format, so remove the pluses):
```diff
 # TODO: add pyparsing to thirdparty folder, update setup.py
+import sys
+sys.path.insert(0,"/usr/local/lib/python2.7/dist-packages/")
 from pyparsing import *
 
 # Enable PackRat for better performance:
```

This makes vipermonkey pull in /usr/local/lib/python2.7/dist-packages/pyparsing.py, which was installed as part of the vipermonkey install, and the tool seems to run fine after that.

**Comment by decalage2** [comment]
I just published a new version of oletools 0.52.3 which does not contain pyparsing anymore. Could you please update oletools with `pip install -U oletools` and tell me if it fixes the issue? Thanks.

**Comment by recvfrom** [comment]
I had messed around a lot with my installation, so I just reverted my VM back to a clean snapshot and did a fresh install.  It appears to work fine, now.  :D
